### PR TITLE
Doc Update

### PR DESCRIPTION
**Pull request by djgagne**
Added jupyter notebooks to docs and install instructions.

### CODE REVIEW

File: docs/source/install.rst [hunk 1]
```diff
@@ -1,6 +1,59 @@
...
+tensorflow, pytorch, or jax. The current version
+of tensorflow (2.15) will downgrade keras 3 to
+keras 2.15 upon installation. If you run into
+this issue, reinstall keras 3 by running the
+following command:
+
+.. code-block:: bash
+    pip install --upgrade keras
+
+
```
Comment (kvrouwenvelder) [inline docs/source/install.rst:52]: recommend adding a 'tutorial' page walking through the notebooks, their purposes, and linking to the notebooks in Sphinx' index

File: docs/source/notebooks/classifier_example.ipynb [hunk 2]
```diff
@@ -0,0 +1 @@
+../../../notebooks/classifier_example.ipynb
```
Comment (kvrouwenvelder) [inline docs/source/notebooks/classifier_example.ipynb:1]: Will you all eventually add the other notebooks w/documentation in Sphinx?

File: docs/source/install.rst [hunk 3]
```diff
@@ -1,6 +1,59 @@
 .. install
 
 Install MILES GUESS
--------------------
+===================
+First, set up a base Python environment on
```
Comment (kvrouwenvelder) [inline docs/source/install.rst:5]: are any comments needed here about supported environments/versions of Python?

File: docs/source/install.rst [hunk 4]
```diff
@@ -1,6 +1,59 @@
...
+keras 2.15 upon installation. If you run into
+this issue, reinstall keras 3 by running the
+following command:
+
+.. code-block:: bash
+    pip install --upgrade keras
+
+
+
+
```
Comment (kvrouwenvelder) [inline docs/source/install.rst:54]: good place to leave troubleshooting notes for users getting started, if any arise. I've also seen packages incorporate an FAQ page into their documentation

**Review by kvrouwenvelder — APPROVED** [review]

File: docs/source/install.rst (hunk 3 above)
Comment (djgagne) [reply docs/source/install.rst:5]: I can add that.

**Pull request merged**